app/pandas.ipynb

```diff
@@ -119,18 +132,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
-   "source": [
-    "# Write your code here"
-   ],
+   "source": "persons[\"Salary\"] = [50000, 60000, 45000]",
```

Comment:
The 'Salary' value for Ryan should be corrected from 45,000 to 450,000 as per the task requirements. Please update the list to `[50000, 60000, 450000]`.